src/cloud/env/variables-global.md

```diff
@@ -16,6 +16,24 @@ stage:
 
 {% include cloud/customize-build-deploy.md %}
 
+### `SCD_USE_BALER`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.3.0 and later
+
+[Baler](https://github.com/magento/baler) is a Magento module that scans your generated JavaScript code and creates an optimized JavaScript bundle. Deploying the optimized bundle to your site can reduce the number of network requests when loading your site and improve page load times.
+
+Set to `true` to run Baler after performing static content deployment.
+
+```yaml
+stage:
+  build:
+    SCD_USE_BALER: true
+```
+
+{:.bs-callout-info}
+You must install and configure the Baler module before using this feature. Because Baler is currently in alpha release, enable this option only on Staging environments.
```

Comment:
```suggestion
You must install and configure the Baler module before using this feature. Because Baler is currently in alpha release, we do not recommend using it in Production environments.
```